### ISSUE TITLE
Provide use cases for monitoring events from WIS2 nodes

### ISSUE THREAD

**Issue by josusky** [wmo-im/wis2-monitoring-alerting#2]
WMEs (WIS2 Monitoring Events) will be primarily used by the WIS2 Global Services but there is no reason why "ordinary" nodes could not use them too, e.g. to report an outage.

**Comment by SimonElliottEUM** [comment]
@josusky I agree with you. We need a mechanism to replace the addressed messages used on the GTS to send service messages. We are working on that in the form of "notices" as a type of WME.
